ui/appui-react/src/appui-react/toolbar/Toolbar.tsx

```diff
@@ -54,6 +55,18 @@ export interface ToolbarProps extends CommonProps, NoChildrenProps {
  * @beta
  */
 export function Toolbar(props: ToolbarProps) {
+  const previewFeatures = usePreviewFeatures();
+  if (previewFeatures.newToolbars) {
+    return <NewToolbar {...props} />;
+  }
+  return <OriginalToolbar {...props} />;
+}
+
+function NewToolbar(_props: ToolbarProps) {
+  return <>WIP</>;
```

Comment:
Do you want to keep that in the PR ? If so, should we add it in the `PreviewFeatureToggle` ?

Reply:
Well, I needed a story with a component that used the `usePreviewFeature` hook to verify the problem w/ a Provider.
I think we can keep it as is for now.